src/common/docker-compose/index.ts

```diff
@@ -672,9 +672,14 @@ export class DockerComposeUtils {
           const state = container_state.State.toLowerCase();
           const health = container_state.Health.toLowerCase();
 
-          const bad_state = state !== 'running';
+          const bad_state = state !== 'running' && container_state.ExitCode !== 0;
```

Comment:
Ah interesting I forgot we treat all non-running states as "bad" - I think this works but I'll test a couple of the weird dev shutdown cases I was using before to make sure

Reply:
Sounds good. Thank you!!